src/InThePopUp.js

```diff
@@ -0,0 +1,33 @@
+import { Component } from 'react';
+import Image from 'react-bootstrap/Image';
+import data from './data.json';
+// import PopUp from './PopUp.js';
+// import Figure from 'react-bootstrap/Figure';
+// import FigureImage from 'react-bootstrap/FigureImage';
+// import FigureCaption from 'react-bootstrap/FigureCaption';
+
+
+class InThePopUp extends Component {
```

Comment:
Feedback here is to consider naming this class something else a bit more specific. What does it represent? Is this going to be the selected horned animal etc. 👍